ti/report/parser/junit/gojunit/ingest.go

```diff
@@ -16,33 +16,40 @@ import (
 	ti "github.com/harness/ti-client/types"
 )
 
+const cypressRootSuiteName = "Root Suite"
+
 // findSuites performs a depth-first search through the XML document, and
 // attempts to ingest any "testsuite" tags that are encountered.
-func findSuites(nodes []xmlNode, suites chan Suite) {
+func findSuites(nodes []xmlNode, suites chan Suite, parentFilename string) {
 	for _, node := range nodes {
 		switch node.XMLName.Local {
 		case "testsuite":
-			suites <- ingestSuite(node)
+			suites <- ingestSuite(node, parentFilename)
+			if node.Attr("name") == cypressRootSuiteName {
```

Comment:
Come up with a better name and don't make it cypress specific
```suggestion
			if node.Attr("name") == someBetterName {
```

Reply:
Done